client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -243,7 +249,8 @@ object CapiModelEnrichment {
       val isShowcase: ContentFilter = content => displayHintExistsWithName("column")(content) ||
         displayHintExistsWithName("showcase")(content) ||
         hasShowcaseImage(content) ||
-        hasShowcaseEmbed(content)
+        hasShowcaseEmbed(content) ||
+        isGallery(content)
```

Comment:
Is this meant to be?

```suggestion
        isPictureContent(content)
```

Reply:
oops - good spot